index.js

```diff
@@ -190,15 +205,22 @@ if (commander.init) {
         console.log(`${data}\nappended in /CHANGELOG.md`);
     }).then(() => {
         // delete JSON changelog files
-        fs.readdirSync(fileDir).forEach(file => {
+        fs.readdirSync(fileDir).forEach((file) => {
             fs.access(`${fileDir}/${file}`, error => {
                 if (!error) {
                     fs.unlinkSync(`${fileDir}/${file}`, (error) => { throw error });
                 } else {
                     throw error;
                 }
             });
-        })
+        });
+    }).then(() => {
+        if (config.autoCommit) {
```

Comment:
I think we should have a special config for merge (eg: `config.autoCommitMerge`)

Reply:
Done, I let you resolve if that suits you